response.proto

```diff
@@ -562,6 +562,9 @@ message Response{
     optional Co2Emission car_co2_emission = 70;
 
     optional StreetNetworkRoutingMatrix sn_routing_matrix = 72;
+
+    // Date time for creating the next kraken call
+    optional uint32 next_request_date_time = 73;
```

Comment:
For the record, as mentioned orally: you can probably use the requested_datetime from journey (if it is correctly filled), so there would be no need for this to be merged in master.